spec/graphql/types/query_type_spec.rb

```diff
@@ -0,0 +1,27 @@
+require "rails_helper"
+
+RSpec.describe "Types::QueryType" do
+  describe "Listings" do
+    let!(:listings) { create_pair(:listing) }
```

Comment:
let! will call create(obj).save() on the test db from the listings fixture and assign it to a variable called listings

this is a method exposed from FactoryBot